corehq/apps/registration/templates/registration/register_new_user.html

```diff
@@ -91,13 +83,11 @@
   <div class="row">
     <div class="col-xs-12">
 
-      {% if is_saas_environment and signup_ux_ab and signup_ux_ab.version == 'on' %}
-        {% include 'registration/partials/start_options.html' %}
-      {% endif %}
+      {% include 'registration/partials/start_options.html' %}
```

Comment:
this will show up on non SAAS environments...I don't think that's what we want

Reply:
Thanks for catching this!